server/middlewares/validateUser.js

```diff
@@ -0,0 +1,38 @@
+import { userSchema } from '../schemas/userSchema';
+/**
+ * This class contains all methods
+ * required to validate
+ * sent data from the UI.
+ */
+class validateUser {
+  /**
+   * This method validates the data sent from the UI.
+   * @param {object} req The user's request.
+   * @param {object} res The response.
+   * @param {object} next pass to next method.
+   * @returns {object} The error and some error messages .
+   */
+  static validateUser(req, res, next) {
```

Comment:
This function is not validating the user. Refactor accordingly.

Reply:
changed it to `validateUserData`

Reply:
Change it to **verifyNotificationChange** instead. Because you're verifying not validating. You can change the file name to match this context as well. You can name the file, **verifyNotificatio**n.

Reply:
Agreed.

Reply:
done